website/general_issues.qmd

```diff
@@ -69,4 +69,13 @@ All wrappers must still be placed inside the `\examples{}` tag.
 
 :::
 
+# Package Size
+
+CRAN package tarballs should be under 5 Mb, with very few exceptions (and these will trigger flags for every future submission, so it's best to minimize the package size). Large datasets can be hosted externally, such as on GitHub, and downloaded when necessary. Consider placing them in a separate data package that requires less frequent updates.
+
```

Comment:
Maybe also add a paragraph about that section of the CRAN policy:

> Source package tarballs should if possible not exceed 10MB. It is much preferred that third-party source software should be included within the package (as e.g. a vendor.tar.xz file) than be downloaded at installation: if this requires a larger tarball a modestly increased limit can be requested at submission.

Something along the lines:
```suggestion
At a upper limit of the package tarball the CRAN policy states 10 Mb. It is recommended to include third-party source software within the package. If that requires larger tarballs, a modestly increased limit can be requested at submission.
```